### PR TITLE
remove noisy logging

### PR DESCRIPTION
**Pull request by dbachelder**

### CODE REVIEW

File: rbridge/rbridge/src/main/java/org/rbridge/RBridgeWorker.java [hunk 1]
```diff
@@ -71,8 +71,6 @@ public JDefinedClass innerRun(Provider<ASTType> astTypeProvider) {
                 JDefinedClass innerBridge = rBridgeImpl._class(PUBLIC | STATIC | FINAL, innerName);
 
                 for (ASTField field : innerType.getFields()) {
-                    log.debug("Mapping " + target + "." + innerName + "." + field.getName() + " = " + id);
```
Comment (johncarl81) [inline rbridge/rbridge/src/main/java/org/rbridge/RBridgeWorker.java]: I kind of like this debug logging... what do you think about added a flag to enable debug logging, effectively turning it off for the default case?

Comment (johncarl81) [inline rbridge/rbridge/src/main/java/org/rbridge/RBridgeWorker.java]: Here's how:
1.  Check for the debug flag (`rbridgeDebug`?) in the `RBridgeModule` like this:
https://github.com/johncarl81/transfuse/blob/master/transfuse/src/main/java/org/androidtransfuse/config/TransfuseAndroidModule.java#L169
2. Replace the `true` constant [here](https://github.com/johncarl81/transfuse/blob/LibraryR/rbridge/rbridge/src/main/java/org/rbridge/RBridgeModule.java#L89) with the debug parameter provided above, like here:
https://github.com/johncarl81/transfuse/blob/master/transfuse/src/main/java/org/androidtransfuse/config/TransfuseAndroidModule.java#L133

**Comment by dbachelder** [comment]
@johncarl81 how about that?

**Comment by johncarl81** [comment]
:+1: You're the man!

**Pull request merged**